includes/admin/class-wp-job-manager-admin.php

```diff
@@ -128,6 +128,21 @@ public function admin_enqueue_scripts() {
 			);
 		}
 
+		if ( 'job_listing' === $screen->id && $screen->is_block_editor() ) { // Check if it's block editor in job post.
```

Comment:
It seems the standard to include the JS files is here, so I included a JS for the job editor here, including a global variable for the promote URL.